extensions/resteasy-reactive/rest-client-reactive/runtime/pom.xml

```diff
@@ -79,7 +79,7 @@
                 <artifactId>quarkus-extension-maven-plugin</artifactId>
                 <configuration>
                     <capabilities>
-                        <provides>io.quarkus.rest.client</provides>
+                        <provides>io.quarkus.rest.client.reactive</provides>
```

Comment:
I think this is going to be a problem for: https://github.com/quarkusio/quarkus/blob/main/extensions/narayana-lra/deployment/src/main/java/io/quarkus/narayana/lra/deployment/NarayanaLRAProcessor.java#L48 .

You need to add a `or` to check if the Reactive REST Client is around.

Reply:
CC @brunobat @radcortez